youtube_dl/extractor/generic.py

```diff
@@ -2874,12 +2874,17 @@ def merge_dicts(dict1, dict2):
         # Look for HTML5 media
         entries = self._parse_html5_media_entries(url, webpage, video_id, m3u8_id='hls')
         if entries:
-            for entry in entries:
-                entry.update({
+            if len(entries) == 1:
+                entries[0].update({
                     'id': video_id,
-                    'title': video_title,
-                })
-                self._sort_formats(entry['formats'])
+                    'title': video_itle})
+                self._sort_formats(entries[0]['formats'])
+            else:
+                for num, entry in enumerate(entries):
+                    entry.update({
+                        'id': video_id,
+                        'title': '%s (%d) ' % (video_title, num+1)})
```

Comment:
`E226 missing whitespace around arithmetic operator`.

Reply:
i see. anyway i removed the +1 in my commit to not get "title (2)", "title (3)", "title (4)" in case of 3 different videos (because you suggested start=1 above). let me know, if you do not aggree.